tests/test_crud.py

```diff
@@ -1,10 +1,14 @@
 import json
 import requests
-from backend.api.main import Datapoint
+import sys
+import re
+import pydantic
+
+sys.path.append("../../n5geh.tools.mqtt-gateway")
```

Comment:
Why the sys.path has to be added?

Reply:
Since setting was not detected by VS code so i added this and then it worked perfectly

Reply:
See above